src/Chapter1.hs

```diff
@@ -632,7 +648,14 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let
+      last1 :: Int -> Int
+      last1 x = mod x 10
+      last2 :: Int -> Int
+      last2 x = mod x 100
```

Comment:
I guess here you need to take `mod` of `div` of the `last1`.

But in this case, you can see that you use both:

```haskell
mod (abs x) 10
div (abs x) 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(lastDiv, last1) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂